pkg/kev/extract.go

```diff
@@ -66,15 +58,6 @@ func extractVolumesLabels(c *ComposeProject, out *composeOverride) {
 	out.Volumes = vols
 }
 
-// extractServiceTypeLabels extracts service type labels into a label's Service.
-func extractServiceTypeLabels(source composego.ServiceConfig, target *ServiceConfig) {
-	if source.Ports == nil {
-		target.Labels.Add(config.LabelServiceType, config.NoService)
-	} else {
-		target.Labels.Add(config.LabelServiceType, config.ClusterIPService)
-	}
-}
-
 // extractDeploymentLabels extracts deployment related into a label's Service.
 func extractDeploymentLabels(source composego.ServiceConfig, target *ServiceConfig) {
```

Comment:
This func is no longer used - can be removed.

Reply:
It's not used by the sub functions were not moved yet so it's here for reference